k6/index.html

```diff
@@ -81,6 +82,27 @@ <h2>検索結果</h2>
           </ul>
        </div>
     </footer>
+
+    <div id="grade-notice">
+       <div class="title">治療成績の算出方法</div>
+       <div>以下「移植あたり生産率」をもって、95%信頼区間を考慮した全国平均との比較をします。</div>
+       <div class="math">$$\text{移植あたり生産率} = \frac{\text{生産分娩数}}{\text{移植総回数}} \times 100$$</div>
+
+       <div>95%信頼区間の計算：</div>
+       <div class="math">
+          $$\text{95%信頼区間} = 1.96 \times \sqrt{\frac{(t - b) \times \alpha^2 + b \times (1 - \alpha)^2}{t}}$$
+          <ul>
+             <li>t: 移植総回数(回)</li>
+             <li>b: 生産分娩数(回)</li>
+             <li>α: 生産分娩数(回) ÷ 移植総回数(回) × 100 (整数値への四捨五入なし)</li>
```

Comment:
> 移植あたり生産率とαは同じものとわかるようにした方がいいですかね。

@sunmoonStern もしかしたら、単純にここ書き換えただけで良いのかも？

```suggestion
             <li>α: 移植あたり生産率</li>
```

Reply:
...と思うじゃないですか。

めちゃめちゃハイコンテキストなんですが、
こども家庭庁のHPでは移植あたり生産率が整数値に丸められている、というのと、
医療機関には、移植あたり生産率の数値がおかしくて、どうも妊娠数(回)を分子にしているところがあったので、
だから、生産分娩数(回) ÷ 移植総回数(回) × 100 (整数値への四捨五入なし)をαとして、
「移植あたり生産率」として定義した方がいいかなと

Reply:
@sunmoonStern こういうことでしょうか？「四捨五入なし」の文言を入れられてないですが

<img width="296" alt="image" src="https://github.com/code-for-future-moms/prototype-k/assets/42468/7fa667e6-4971-4c66-8d31-36e49a619634">

Reply:
こうか
<img width="286" alt="image" src="https://github.com/code-for-future-moms/prototype-k/assets/42468/eb7a3662-1bc4-4a26-81ed-3d2fc3ae233f">

Reply:
```diff
diff --git a/k6/index.html b/k6/index.html
index 542feb8..32b661f 100644
--- a/k6/index.html
+++ b/k6/index.html
@@ -85,16 +85,16 @@
 
     <div id="grade-notice">
        <div class="title">治療成績の算出方法</div>
-       <div>以下「移植あたり生産率」をもって、95%信頼区間を考慮した全国平均との比較をします。</div>
-       <div class="math">$$\text{移植あたり生産率} = \frac{\text{生産分娩数}}{\text{移植総回数}} \times 100$$</div>
+       <div>95%信頼区間を考慮した全国平均との比較をします。</div>
 
-       <div>95%信頼区間の計算：</div>
        <div class="math">
           $$\text{95%信頼区間} = 1.96 \times \sqrt{\frac{(t - b) \times \alpha^2 + b \times (1 - \alpha)^2}{t}}$$
           <ul>
-             <li>t: 移植総回数(回)</li>
-             <li>b: 生産分娩数(回)</li>
-             <li>α: 生産分娩数(回) ÷ 移植総回数(回) × 100 (整数値への四捨五入なし)</li>
+             <li>t = 移植総回数(回)</li>
+             <li>b = 生産分娩数(回)</li>
+             <li>
+                <div class="math">$$\text{α} = \frac{\text{生産分娩数}}{\text{移植総回数}} \times 100\ (整数値への四捨五入なし)$$</div>
+             </li>
           </ul>
        </div>
 
diff --git a/k6/style.css b/k6/style.css
index d9675ae..e336f90 100644
--- a/k6/style.css
+++ b/k6/style.css
@@ -66,6 +66,10 @@ h3.title {
   font-weight: bold;
 }
 
+.MathJax {
+  text-align: left !important;
+}
+
 #grade-notice .math {
   font-size: 90%;
 }
```

Reply:
もろもろ試行錯誤ありがとうございます。いろいろやってもらってあれなんですけど、
なんというか、いまcommitしてもらっている記述の方がよい部分があると思います。
いまαといっている部分が一般的に言われる移植あたり生産率と(丸めを無視すれば)同じ数値だとわかるし、
治療成績のいい悪いをジャッジしているのが移植あたり妊娠率などではなく、移植あたり生産率だとわかるので。

ですので自分の提案は前言ったように

> 最初の移植あたり生産率の式で
> 移植あたり生産率(α)
> とか? 🤔

になります。

Reply:
@sunmoonStern こんな感じでどうでしょう！

<img width="311" alt="image" src="https://github.com/code-for-future-moms/prototype-k/assets/42468/9a23be6b-7ca5-4ce6-a243-da0f804c6c01">

Reply:
@ktakayama バッチリです！こちらでお願いします 🙇🏼‍♀️